### PR TITLE
Tlcpm 309 - mop up

### PR DESCRIPTION
**Pull request by dlhaines**
Incorporates a number of cleanups, including revised gems, build script, better testing organization, non default config file, case changes in gem name for linux vs osx.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,9 +1,13 @@
-PATH
-  remote: /Users/dlhaines/dev/GITHUB/dlh-umich.edu/FORKS/google_groups_gem
+GIT
+  remote: https://github.com/dlhaines/google_groups_gem.git
+  revision: c64f3f986031b3eec485ffaa465f09fe05823f8d
```
Comment (zqian) [inline Gemfile.lock:3]: In case of new revision of google_groups_gem, will this file be updated?

File: test/test_members.rb [hunk 2]
```diff
@@ -38,7 +13,7 @@ def app
     Sinatra::Application
   end
 
-  TEST_DOMAIN='discussions-dev.its.umich.edu'
+  #TEST_DOMAIN='discussions-dev.its.umich.edu'
```
Comment (zqian) [inline test/test_members.rb:16]: can remove this line since it is already defined in test_helper.rb

File: test/test_app_status.rb [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+class StatusAppGroupsTest < Minitest::Test
+
+  include Rack::Test::Methods
+  include TestHelper
+
+  def app
+    Sinatra::Application
+  end
+
+  TEST_DOMAIN='discussions-dev.its.umich.edu'
```
Comment (zqian) [inline test/test_app_status.rb:14]: the "TEST_DOMAIN" value is defined in couple of places. Can those definitions be combined?

File: Gemfile.lock (hunk 1 above)
Comment (dlhaines) [reply Gemfile.lock:3]: Yes.  This done automatically when running bundle install to update the dependencies.

File: test/test_members.rb (hunk 2 above)
Comment (dlhaines) [reply test/test_members.rb:16]: I'll add that to next round of test cleanup.

**Pull request merged**